src/wrappers/gcc_wrapper.hpp

```diff
@@ -31,12 +31,17 @@ class gcc_wrapper_t : public program_wrapper_t {
   bool can_handle_command() override;
 
 protected:
+  void resolve_args() override;
+  string_list_t parse_args(const string_list_t& args);
+  string_list_t parse_response_file(const std::string& filename);
```

Comment:
These should be private (put them in a `priavet:` section below `m_resolved_args`).

Reply:
Will do.